package.json

```diff
@@ -28,7 +30,9 @@
 		"webpack-cli": "^4.10.0"
 	},
 	"scripts": {
-		"watch": "webpack --watch --mode development",
+		"watch:src": "webpack --watch --mode development",
+		"watch:dist": "mv3-hot-reload",
+		"dev": "concurrently yarn:watch:*",
```

Comment:
what the heck does the thing also not allow the npm script to be named `"watch"`???

Reply:
nah i just stole this from them but i'm bringing `watch` back so we don't have to think about running a diff command lol

Reply:
i mean idc either way, but does it not work with `"watch"`, `"dev:webpack"`, and `"dev:mv3"`?

Reply:
oh oop i think it does